### DISCUSSION TITLE
App UI Suggestions

### DISCUSSION THREAD

**Discussion by aleksey-saenko** [aleksey-saenko/MusicRecognizer#34] [Ideas]
If you have any small suggestions or feedback regarding the app UI, please feel free to share them in this discussion.
For bigger ideas on redesigning the project, you can create a separate topic for discussions. If you encounter any bugs, please report them as issues.
Please provide detailed information about what specific changes you would like to see and why, and if possible, include screenshots to illustrate your points. Please keep in mind that everyone's opinions, including yours and those of other users and developers, may vary. While I will make an effort to consider your input, I cannot guarantee that every suggestion will be implemented in the app.

Looking forward to your thoughts and ideas!

**Comment by ltguillaume** [comment]
I'd really like to see the date of recognition right in the library. In the list layout, there's anough room for it anyway.

**Comment by aleksey-saenko** [comment]
@ltguillaume It would look something like this:

<img src='https://github.com/aleksey-saenko/MusicRecognizer/assets/109096599/84d28be0-5cdc-4bde-b981-45cd02531edf' width='200'>
<img src='https://github.com/aleksey-saenko/MusicRecognizer/assets/109096599/442b05be-f38a-4658-ad0b-446040d0d438' width='200'>
<img src='https://github.com/aleksey-saenko/MusicRecognizer/assets/109096599/f08ad4e3-f605-48cd-9234-78623434cd07' width='200'>

Honestly, in my opinion, the version without a date appears cleaner and neater. Although I understand that a quick glance at the date can also be helpful. In order to ensure a comprehensive viewpoint, let's collect more feedback from other users.

**Reply by ltguillaume** [reply]
The text color could perhaps be a bit lighter, or semi-transparent, to make it less "invasive"?

**Reply by aleksey-saenko** [reply]
More transparency:

<img src='https://github.com/aleksey-saenko/MusicRecognizer/assets/109096599/ce7b01aa-e62e-4172-820c-fea932850ebd' width='200'>
<img src='https://github.com/aleksey-saenko/MusicRecognizer/assets/109096599/f43c9947-2d04-40f6-a656-a13cbdf33090' width='200'>

**Reply by ltguillaume** [reply]
Looks good to me :)

The date format on the right, even though it takes up a bit more horizontal space, feels less invasive than just "a block of numbers" (left), imho. Not sure if you show both because you choose which date format will be used, or because both can occur depending on locale settings, though.

**Reply by aleksey-saenko** [reply]
I agree, the latest version doesn't look so bad for the English locale. For the first screen, `FormatStyle.SHORT` is used, and for the second `FormatStyle.MEDIUM`. However, for Czech and Slovak locales, only numbers are displayed in the last mode (last screen). In general, we should assume that the format can vary greatly depending on the locale.

**Reply by ltguillaume** [reply]
Yeah, that's true. Either way, I assume this is a change people get used to quickly, plus it's really useful to quickly look up the song(s) recognized that one time in that one pub. Without it, it's pretty much impossible to do so.

**Reply by aleksey-saenko** [reply]
Date display has been added in the latest version. The time-month-year format is selected depending on the current date.

**Reply by ltguillaume** [reply]
Great, thank you so much!
